lib/p2p/Peer.ts

```diff
@@ -292,16 +301,25 @@ class Peer extends EventEmitter {
   /**
    * Fulfill awaiting requests response.
    */
-  private fulfillResponse = (packetType: PacketType): PendingResponseEntry | null => {
-    const entry = this.responseMap.get(packetType);
+  private fulfillResponse = (key: string, packet: Packet): boolean => {
+    if (!key) {
```

Comment:
Are you only checking for empty strings here or also for null values? I think if we have a check like this, we can let `key` be `string | null` and not put the `!` operator when passing parameters to this function.

Reply:
Good idea. Fixed, using `undefined` instead.